### PR TITLE
Shm config

### PR DESCRIPTION
**Pull request by at88mph**

### CODE REVIEW

**Review by brianmajor — COMMENTED** [review]
@at88mph - Why the cavern update in this PR?  Are you trying to get helm/master up-to-date with things that weren't committed before?

**Comment by at88mph** [comment]
The Chart is released, but the Chart YAML file wasn't pushed.  This is just bringing `master` up to date with what's released.

**Comment by brianmajor** [comment]
I see.  The process should probably be getting the changes reviewed and PR accepted before release (chart and image).

File: deployment/helm/skaha/skaha-config/launch-desktop.yaml [hunk 1]
```diff
@@ -89,6 +89,8 @@ spec:
           protocol: TCP
           name: novnc-port
         volumeMounts:
+        - mountPath: /dev/shm
+          name: dshm
         - mountPath: "/etc/passwd"
           name: etc-passwd
           subPath: passwd
```
Comment (brianmajor) [inline deployment/helm/skaha/skaha-config/launch-desktop.yaml]: Desktop won't need extra shared memory (not user facing) so we should take it out of here.

File: deployment/k8s-config/kustomize/base/skaha/config/launch-desktop.yaml [hunk 2]
```diff
@@ -72,6 +72,8 @@ spec:
           protocol: TCP
           name: novnc-port
         volumeMounts:
+        - mountPath: /dev/shm
+          name: dshm
```
Comment (brianmajor) [inline deployment/k8s-config/kustomize/base/skaha/config/launch-desktop.yaml:76]: Again, not needed by the desktop

**Review by brianmajor — CHANGES_REQUESTED** [review]
The changes should to be made to kustomize/overlays/keel-dev too...

**Pull request closed without merging**